<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="EPBYMINW5904" ignored="0" name="All_tests_parallel" tests="30" failures="6" timestamp="18 Dec 2017 07:23:46 GMT" time="0.098" errors="0">
  <testcase name="sqrtFromValue" time="0.0" classname="test.testng.SqrtDoubleTest"/>
  <testcase name="oneValueMultAnother" time="0.0" classname="test.testng.MultDoubleTest">
    <failure type="java.lang.AssertionError" message="expected [14.25] but found [14.0]">
      <![CDATA[java.lang.AssertionError: expected [14.25] but found [14.0]
	at test.testng.MultDoubleTest.oneValueMultAnother(MultDoubleTest.java:26)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:748)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- oneValueMultAnother -->
  <testcase name="sqrtFromValue" time="0.0" classname="test.testng.SqrtDoubleTest">
    <failure type="java.lang.AssertionError" message="expected [NaN] but found [10.0]">
      <![CDATA[java.lang.AssertionError: expected [NaN] but found [10.0]
	at test.testng.SqrtDoubleTest.sqrtFromValue(SqrtDoubleTest.java:26)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- sqrtFromValue -->
  <testcase name="oneValuePlusAnother" time="0.0" classname="test.testng.SumLongTest"/>
  <testcase name="checkDivisionByZeroThrowsException" time="0.016" classname="test.testng.DivLongTest"/>
  <testcase name="sinFromValue" time="0.016" classname="test.testng.SinDoubleTest"/>
  <testcase name="ctgFromValue" time="0.016" classname="test.testng.CtgDoubleTest">
    <failure type="java.lang.AssertionError" message="expected [0.642] but found [0.7615941559557649]">
      <![CDATA[java.lang.AssertionError: expected [0.642] but found [0.7615941559557649]
	at test.testng.CtgDoubleTest.ctgFromValue(CtgDoubleTest.java:22)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- ctgFromValue -->
  <testcase name="oneValueMinusAnother" time="0.016" classname="test.testng.SubDoubleTest"/>
  <testcase name="NegativeValueCheck" time="0.016" classname="test.testng.NegativeLongTest"/>
  <testcase name="NegativeValueCheck" time="0.0" classname="test.testng.NegativeLongTest"/>
  <testcase name="NegativeValueCheck" time="0.0" classname="test.testng.NegativeLongTest"/>
  <testcase name="sinFromValue" time="0.0" classname="test.testng.SinDoubleTest"/>
  <testcase name="oneValueDivAnother" time="0.0" classname="test.testng.DivLongTest"/>
  <testcase name="oneValueDivAnother" time="0.0" classname="test.testng.DivLongTest">
    <failure type="java.lang.AssertionError" message="Invalid result of operation expected [-1.5] but found [-1.0]">
      <![CDATA[java.lang.AssertionError: Invalid result of operation expected [-1.5] but found [-1.0]
	at test.testng.DivLongTest.oneValueDivAnother(DivLongTest.java:25)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- oneValueDivAnother -->
  <testcase name="oneValuePlusAnother" time="0.016" classname="test.testng.SumLongTest"/>
  <testcase name="oneValuePlusAnother" time="0.0" classname="test.testng.SumLongTest"/>
  <testcase name="sqrtFromValue" time="0.0" classname="test.testng.SqrtDoubleTest"/>
  <testcase name="sqrtFromValue" time="0.0" classname="test.testng.SqrtDoubleTest"/>
  <testcase name="oneValuePlusAnother" time="0.0" classname="test.testng.SumLongTest"/>
  <testcase name="sqrtFromValue" time="0.0" classname="test.testng.SqrtDoubleTest"/>
  <testcase name="oneValueDivAnother" time="0.0" classname="test.testng.DivLongTest">
    <failure type="java.lang.AssertionError" message="Invalid result of operation expected [0.1] but found [0.0]">
      <![CDATA[java.lang.AssertionError: Invalid result of operation expected [0.1] but found [0.0]
	at test.testng.DivLongTest.oneValueDivAnother(DivLongTest.java:25)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- oneValueDivAnother -->
  <testcase name="sinFromValue" time="0.016" classname="test.testng.SinDoubleTest"/>
  <testcase name="oneValueMinusAnother" time="0.016" classname="test.testng.SubDoubleTest"/>
  <testcase name="oneValueMinusAnother" time="0.015" classname="test.testng.SubDoubleTest"/>
  <testcase name="oneValueMultAnother" time="0.015" classname="test.testng.MultDoubleTest"/>
  <testcase name="oneValueMultAnother" time="0.0" classname="test.testng.MultDoubleTest"/>
  <testcase name="oneValueDivAnother" time="0.015" classname="test.testng.DivLongTest"/>
  <testcase name="oneValueMultAnother" time="0.0" classname="test.testng.MultDoubleTest">
    <failure type="java.lang.AssertionError" message="expected [0.35] but found [0.0]">
      <![CDATA[java.lang.AssertionError: expected [0.35] but found [0.0]
	at test.testng.MultDoubleTest.oneValueMultAnother(MultDoubleTest.java:26)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:748)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- oneValueMultAnother -->
  <testcase name="oneValueMinusAnother" time="0.0" classname="test.testng.SubDoubleTest"/>
  <testcase name="oneValueMinusAnother" time="0.0" classname="test.testng.SubDoubleTest"/>
</testsuite> <!-- All_tests_parallel -->
